addons/pronelauncher/CfgMovesMaleSdr.hpp

```diff
@@ -0,0 +1,239 @@
+                "AmovPpneMstpSrasWpstDnon", 0.02
+            };
+        };
+        class ProneRifle_To_ProneLauncher: ProneLauncher_To_ProneRifle {
+            speed = 0.76;
+            duty = 0.6;
+            interpolateFrom[] = {
+                "AmovPpneMstpSrasWrflDnon", 0.02
+            };
+            interpolateTo[] = {
+                "AmovPpneMrunSrasWlnrDf", 0.015,
+                "AmovPpneMrunSrasWlnrDr", 0.015,
+                "AmovPpneMrunSrasWlnrDl", 0.015,
+                "ACE_LauncherProne", 0.015
+            };
+        };
+        class PronePistol_To_ProneLauncher: ProneLauncher_To_ProneRifle {
+            speed = 0.76;
+            duty = 0.6;
+            interpolateFrom[] = {
+                "AmovPpneMstpSrasWpstDnon", 0.015
+            };
+            interpolateTo[] = {
+                "AmovPpneMrunSrasWlnrDf", 0.015,
+                "AmovPpneMrunSrasWlnrDr", 0.015,
+                "AmovPpneMrunSrasWlnrDl", 0.015,
+                "ACE_LauncherProne", 0.015
+            };
+        };
+        class ProneCivil_To_ProneLauncher: ProneLauncher_To_ProneRifle {
+            speed = 0.76;
+            duty = 0.6;
+            interpolateFrom[] = {
+                "AmovPpneMstpSnonWnonDnon", 0.015
+            };
+            interpolateTo[] = {
+                "AmovPpneMrunSrasWlnrDf", 0.015,
+                "AmovPpneMrunSrasWlnrDr", 0.015,
+                "AmovPpneMrunSrasWlnrDl", 0.015,
+                "ACE_LauncherProne", 0.015
+            };
+        };
+        class AmovPercMstpSrasWlnrDnon_AmovPpneMstpSrasWlnrDnon: TransAnimBase {
+            mask = "weaponSwitching";
+        };
+        class AmovPpneMstpSrasWlnrDnon_AmovPknlMstpSrasWlnrDnon: TransAnimBase {
+            blockMobileSwitching = 0;
+            ConnectTo[] = {};
+            InterpolateTo[] += {"AmovPknlMstpSrasWlnrDnon", 0.02};
+        };
+        class LauncherProne_Reload_Start: AmovPpneMrunSrasWlnrDl {
+            actions = "ACE_LauncherProneActions";
+            speed = 0.7375;
+            duty = 0.6;
+            disableWeapons = true;
+            disableWeaponsLong = true;
+            enableMissile = false;
+            canPullTrigger = false;
+            interpolateFrom[] = {
+                "ACE_LauncherProne", 0.02
+            };
+            interpolateTo[] = {
+                "LauncherProne_Reload_Mid", 0.005
+            };
+        };
+        class LauncherProne_Reload_Mid: AmovPpneMrunSrasWlnrDr {
+            actions = "ACE_LauncherProneActions";
+            speed = 0.7375;
+            duty = 0.6;
+            disableWeapons = true;
+            disableWeaponsLong = true;
+            enableMissile = false;
+            canPullTrigger = false;
+            interpolateTo[] = {
+                "LauncherProne_Reload_End", 0.005
+            };
+        };
+        class LauncherProne_Reload_End: AmovPpneMrunSrasWlnrDf {
+            actions = "ACE_LauncherProneActions";
+            speed = 0.7375;
```

Comment:
Out of interest, where are these numbers from? They seem pretty random.

Reply:
They are, placeholders for when a nicer animation is found.

Reply:
So what would happen if you just set them to 1 or 0?